### PR TITLE
[dokuwiki] Add dokuwiki helm chart

### PR DESCRIPTION
**Pull request by psych0d0g**
lets try this again without the clutter ;)

Description of the change
added parameterized helm chart to deploy dokuwiki (a small and lightweight php based wiki system using filesystem storage by default)

Benefits

More deploy-able products ;)

Possible drawbacks

no known limitations yet

### CODE REVIEW

File: charts/stable/dokuwiki/values.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+
+# -- environment variables. See [image docs](https://docs.kanboard.org/en/latest/admin_guide/docker.html#environment-variables)
+# and [application docs](# https://docs.kanboard.org/en/latest/admin_guide/config_file.html) for more details.
+# @default -- See below (only deviations from the default settings are specified)
+env:
+  # Application Configuration
+  # -- Application Timezone
+  TZ: Europe/London
```
Comment (bjw-s) [inline charts/stable/dokuwiki/values.yaml:21]: We try to default the TZ variable to UTC.

```suggestion
  TZ: UTC
```

**Review by bjw-s — CHANGES_REQUESTED** [review]
Looking good, thanks for the PR! I've added a tiny change request :-)

**Comment by psych0d0g** [comment]
> Looking good, thanks for the PR! I've added a tiny change request :-)

thats okay, down to 1 change request for the second pull request is okay i guess, aiming to perfection, so hopefully the next one comes without any changes requested and can just be merged :D

**Review by bjw-s — APPROVED** [review]

**Comment by pagong** [comment]
Thanks, for creating this helm chart. 
This values.yaml file is much smaller and less confusing than the one for the bitnami chart for dokuwiki.

I'm using k3s-1.22.5 with traefik-2.5 as ingress router. Works perfectly.  :+1:

**Pull request merged**